packages/builder/src/pages/builder/portal/apps/_FirstAppOnboarding/ExampleApp.svelte

```diff
@@ -0,0 +1,254 @@
+<script>
+  export let name = ""
+  export let showData = false
+
+  const rows = [
+    {
+      firstName: "Julie",
+      lastName: "Jimenez",
+      email: "julie.jimenez@example.com",
+      address: "4250 New Street",
+      city: "Stevenage",
+      postcode: "EE32 3SE",
+      phone: "01754 13523",
+    },
+    {
+      firstName: "Mandy",
+      lastName: "Clark",
+      email: "mandy.clark@example.com",
+      address: "8632 North Street",
+      city: "Hereford",
+      postcode: "GT81 7DG",
+      phone: "016973 32814",
+    },
+    {
+      firstName: "Holly",
+      lastName: "Carroll",
+      email: "holly.carroll@example.com",
+      address: "5976 Springfield Road",
+      city: "Edinburgh",
+      postcode: "Y4 2LH",
+      phone: "016977 73053",
+    },
+    {
+      firstName: "Francis",
+      lastName: "Castro",
+      email: "francis.castro@example.com",
+      address: "3970 High Street",
+      city: "Wells",
+      postcode: "X12 6QA",
+      phone: "017684 23551",
+    },
+  ]
+</script>
+
+<div tabindex="-1" class="exampleApp">
```

Comment:
I chose to just reimplement a dummy version of the app, drawer and table here. There were issues with themes and such and this seemed simpler.

Reply:
Yep that's totally fine. Makes sense to hardcode everything here to keep it as safe as possible.